oci/table_oci_logging_log_group.go

```diff
@@ -167,6 +167,13 @@ func listLoggingLogGroups(ctx context.Context, d *plugin.QueryData, _ *plugin.Hy
 	for pagesLeft {
 		response, err := session.LoggingManagementClient.ListLogGroups(ctx, request)
 		if err != nil {
+			plugin.Logger(ctx).Error("listLoggingLogGroups", "error_ListLogGroups", err)
+			if ociErr, ok := err.(common.ServiceError); ok {
+				if ociErr.GetCode() == "InvalidParameter" {
+					return logging.ListLogGroupsResponse{}, nil
```

Comment:
Please move the changes made in the `.go` files into a separate PR since they refer to a separate bug altogether